invisible_cities/cities/detsim_simulate_photons_test.py

```diff
@@ -0,0 +1,63 @@
+import os
+import numpy  as np
+import tables as tb
+import pandas as pd
+from pytest import fixture
+
+import invisible_cities.database.load_db as db
+
+from invisible_cities.io.mcinfo_io import read_mchits_df
+
+from invisible_cities.cities.detsim_simulate_photons import *
+
+## same as in detsim_simulate_electrons_test
+@fixture(params = ["Kr83_nexus_v5_03_00_ACTIVE_7bar_3evts.MCRD.h5"])
+def nexusMC_filename(request, ICDATADIR):
+    return os.path.join(ICDATADIR, request.param)
```

Comment:
I think this file is already in a session scope fixture in `invisible_cities/conftest.py` , I'm not sure you gain much by redefining it